File: articles/grace-hopper-compilers-and-the-humanization-of-programming/java/CompilerPipeline.java
import java.util.Arrays;

public class CompilerPipeline {
    public static void main(String[] args) {
        String source = "ADD PAYROLL-TOTAL TO TAX-BASE";
        String[] tokens = source.split("\\s+");
        System.out.println("source=" + source);
        System.out.println("tokens=" + Arrays.toString(tokens));
        System.out.println("target_code=machine-specific instruction sequence");
    }
}

File: articles/grace-hopper-compilers-and-the-humanization-of-programming/c/compiler_pipeline.c
#include <stdio.h>

int main(void) {
    const char *source = "ADD PAYROLL-TOTAL TO TAX-BASE";
    printf("source=%s\n", source);
    printf("tokens=[ADD, PAYROLL-TOTAL, TO, TAX-BASE]\n");
    printf("target_code=machine-specific instruction sequence\n");
    return 0;
}
